templates/contract-deploy/update-ger.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
nit: we can simplify this using `polycli ulxly deposit-new` imo - I'll take care of that in a follow up PR